src/providers/firebase-messaging.ts

```diff
@@ -14,7 +14,7 @@ export class FirebaseMessaging implements ProviderInterface {
 	constructor(certPath: string) {
 		this.certPath = certPath;
```

Comment:
please change certPath to just be `cert`

Reply:
Updated